src/app/containers/StoryPromo/index.jsx

```diff
@@ -79,6 +79,7 @@ const StoryPromoContainer = ({
   dir,
   displayImage,
   displaySummary,
+  isRecommendation,
```

Comment:
You've added a new prop here. Can you please add a unit test and a story in Storybook? 
```
src/app/containers/StoryPromo/index.test.jsx
src/app/containers/StoryPromo/index.stories.jsx
```